portfolio/src/main/webapp/art.html

```diff
@@ -0,0 +1,119 @@
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Rock+Salt" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Cinzel+Decorative" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Poiret+One&display=swap" rel="stylesheet">
+    <link href="style.css" rel="stylesheet" type="text/css" />
+
+
+  </head>
+  <body>
+
+
+    <div id="content">
+
+
+    <div id="logo"><em><h1 id="first"><div id="altletters"><span id="c1">C</span><span id="c12">hi</span><span id="c2">&#10037;a<span id="c23">&#10037;</span></span><span id="c3">m</span><span id="c4">a</span><span id="c45">&#10037;</span><span id="c5">k</span><span id="c6">a</span></div></h1></em></div>
+    <h3 id="subhead"> &#9830; Chiamaka  (Chee-uh-mok-uh)  Chukwuka  (Chew-kwoo-kah). &#9830;</h3>
+    <script src="script.js"></script>
+
+    
+    </div>
+    <head><title>Artwork</title></head>
+    <body>
+      <link href="https://fonts.googleapis.com/css?family=Playfair+Display+SC" rel="stylesheet">
+      <link href="https://fonts.googleapis.com/css?family=Homemade+Apple" rel="stylesheet">
+      <link href="https://fonts.googleapis.com/css?family=Monsieur+La+Doulaise" rel="stylesheet">    
+      <link href="style.css" rel="stylesheet" type="text/css" />
+      <header>
+        <ul class="navigation">
+            <a href="./index.html"><li>HOME</li></a>
+            <a href="./projects.html"><li>PROJECTS</li></a>
+            <a href="./art.html"><li>ARTWORK</li></a>
+            <a href="./aboutme.html"><li>ABOUT ME</li></a>
+        </ul>
+      </header>
+    <div class ="artbody">
+
+        <br></br>
+        <br></br>
+        <br></br>
+
+
+        <div class="separator" style="clear: both; text-align: center;">
+        <img border="0" height="900" src="./images/feet.jpg"  width="620" />
+        </div>
+        <p id="desc"> "NOSTALGIA"</p> 
+        <br></br>
+        <br></br>
+        <br></br>
+
+
+
+        <div class="separator" style="clear: both; text-align: center;">
+        <img border="0" height="900" src="./images/ceiling.jpg"  width="920" />
+        </div>
+        <p id="desc"> "ARCHITECTURE CARVING"</p> 
+        <br></br>
+        <br></br>
+        <br></br>
+
+
+
+        <div class="separator" style="clear: both; text-align: center;">
+        <img border="0" height="600" src="./images/overgrowth.jpg"  width="850" />
+        </div>
+        <p id="desc"> "OVERGROWTH"</p> 
+        <br></br>
+        <br></br>
+        <br></br>
+
+
+
+        <div class="separator" style="clear: both; text-align: center;">
+        <img border="0" height="660" src="./images/portrait.jpg"  width="940" />
+        </div>
+        <p id="desc"> "PORTRAIT"</p> 
+        <p> padding</p>
+        <p> padding</p>
```

Comment:
You still have a lot of these in your html files

Reply:
Resolving because I this issue is now fixed